src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -81,6 +84,21 @@ public void updatePerson(ReadOnlyPerson target, ReadOnlyPerson editedPerson)
         indicateAddressBookChanged();
     }
 
+    @Override
+    public void changeTag(Tag oldTag, Tag newTag) throws PersonNotFoundException, DuplicatePersonException {
```

Comment:
This doesn't seems like changing tag for given person, it seems like you're changing tag for entire addressbook, please double check and update your header comment correspondingly